apps/api/src/server/applications/sector/__tests__/get-sectors.test.js

```diff
@@ -89,3 +91,87 @@ describe('Get sectors', () => {
 		});
 	});
 });
+
+test('test if sector cache is working', async () => {
+	nodeCache.flushAll();
+
+	const cacheSector = {
+		id: 3,
+		name: 'cache test name',
+		abbreviation: 'CC',
+		displayNameEn: 'cache test name en',
+		displayNameCy: 'cache test name cy'
+	};
+
+	setCache('', [cacheSector]);
```

Comment:
How come we are setting cache to an empty string here?

Reply:
I can't remember - @LetsMakeItTechnical - what was the reason?